src/webhook/handlers/initState.js

```diff
@@ -3,25 +3,23 @@ import { t } from 'ttag';
 import gql from 'src/lib/gql';
 import {
   createPostbackAction,
-  isNonsenseText,
   ellipsis,
-  ARTICLE_SOURCES,
+  createAskArticleSubmissionConsentReply,
+  createSuggestOtherFactCheckerReply,
 } from './utils';
 import ga from 'src/lib/ga';
 
 const SIMILARITY_THRESHOLD = 0.95;
 
 export default async function initState(params) {
-  let { data, state, event, issuedAt, userId, replies, isSkipUser } = params;
+  let { data, state, event, userId, replies, isSkipUser } = params;
 
   // Track text message type send by user
   const visitor = ga(userId, state, event.input);
-  visitor.event({ ec: 'UserInput', ea: 'MessageType', el: 'text' });
+  visitor.event({ ec: 'UserInput', ea: 'MessageType', el: event.message.type });
```

Comment:
If `process.env.IMAGE_MESSAGE_ENABLED` is not true, image messages will not be tracked.
Maybe we can extract this along with [index.js L168](https://github.com/cofacts/rumors-line-bot/blob/7a93a085e75e9d3ac1bcd8018b4a1625e4353bdf/src/webhook/index.js#L168), [index.js L179](https://github.com/cofacts/rumors-line-bot/blob/7a93a085e75e9d3ac1bcd8018b4a1625e4353bdf/src/webhook/index.js#L179) to the beginning of [process message](https://github.com/cofacts/rumors-line-bot/blob/7a93a085e75e9d3ac1bcd8018b4a1625e4353bdf/src/webhook/index.js#L83).